client/lobby/OptionsTabBase.cpp

```diff
@@ -407,8 +407,14 @@ void OptionsTabBase::recreate()
 	}
 
 	if(auto buttonCheatAllowed = widget<CToggleButton>("buttonCheatAllowed"))
+	{
 		buttonCheatAllowed->setSelectedSilent(SEL->getStartInfo()->extraOptionsInfo.cheatsAllowed);
+		buttonCheatAllowed->setInputEnabled(SEL->screenType != ESelectionScreen::loadGame);
```

Comment:
On a similar note - why block at all? Do we need to block this functionality in saves for some reason?

Reply:
Because after starting the game the setting is currently not changeable. You have to decide at the map beginning.

I don't want to disable the tab because you may want to look how it's configured. But changing the values after game start doesn't make anything.

May change this behavior in the future.